templates/settings/store-form.php

```diff
@@ -338,25 +380,74 @@
 
         $('#dokan-store-time-enable').trigger('change');
 
+        $( '.dokan-form-group-container' ).each( function(e) {
+            $(this).find( '.dokan-form-group' ).each( function (e) {
+                let selectedWorkingStatus = $(this).find( '.dokan-on-off option[selected]' ).length;
+
+                if ( ! selectedWorkingStatus ) {
+                    $(this).find( '.dokan-on-off option[value="close"]' ).attr( 'selected', 'selected' );
+                }
+            } );
+        } );
+
+        // Show & hide our opening, closing time fields by using this change event.
         $( '.dokan-on-off' ).on( 'change', function() {
             var self = $(this);
 
             if ( self.val() == 'open' ) {
                 self.closest('.dokan-form-group').find('.time').css({'visibility': 'visible'});
-                self.closest('.store-open-close').find('.dokan-w6').addClass('dokan-text-left');
-                self.closest('.store-open-close').find('.dokan-w6').css({'width': '50%'});
+                self.closest('.dokan-form-group').find('.opening-time').val('');
+                self.closest('.dokan-form-group').find('.closing-time').val('');
             } else {
                 self.closest('.dokan-form-group').find('.time').css({'visibility': 'hidden'});
                 self.closest('.store-open-close').find('.dokan-w6').removeClass('dokan-text-left');
                 self.closest('.store-open-close').find('.dokan-w6').css({'width': 'auto'});
             }
 
         } );
+
+        // Set timepicker jquery here.
         $( '.time .dokan-form-control' ).timepicker({
             scrollDefault: 'now',
-            timeFormat: '<?php echo addcslashes( esc_attr( get_option( 'time_format' ) ), '\\' ); ?>',
-            step: <?php echo 'h' === strtolower( get_option( 'time_format' ) ) ? '60' : '30'; ?>
+            timeFormat: '<?php echo addcslashes( esc_attr( wc_time_format() ), '\\' ); ?>',
+            step: <?php echo 'h:i' === strtolower( wc_time_format() ) ? '60' : '30'; ?>,
         });
+
+        // Update closing time on focus when previously setted up closing time field.
+        $( '.dokan-form-group-container' ).on( 'focus', '.dokan-form-control',  function(e) {
+            e.stopPropagation();
+
+            let self       = $(this),
```

Comment:
use, like `$( this )`
and
make all of the variable with `const` as, you wouldn't change these later.